app/views/home/failure.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="py1 mb3 center">
+  <h2>Authentication from Login.gov was not successful! :(</h2>
```

Comment:
Can we get failure messages from omniauth?  It think its good to display them here.. but we can just put them in the logs if you prefer.